src/app/qtlocalpeer/qtlockedfile_win.cpp

```diff
@@ -70,166 +70,191 @@
 
 #include <QFileInfo>
 
-#define MUTEX_PREFIX "QtLockedFile mutex "
-// Maximum number of concurrent read locks. Must not be greater than MAXIMUM_WAIT_OBJECTS
-#define MAX_READERS MAXIMUM_WAIT_OBJECTS
+#include "base/global.h"
 
-#define QT_WA(unicode, ansi) unicode
+// Maximum number of concurrent read locks. Must not be greater than MAXIMUM_WAIT_OBJECTS
+const int MAX_READERS = MAXIMUM_WAIT_OBJECTS;
 
-Qt::HANDLE QtLockedFile::getMutexHandle(int idx, bool doCreate)
+Qt::HANDLE QtLockedFile::getMutexHandle(const int idx, const bool doCreate)
 {
-    if (mutexname.isEmpty()) {
+    if (m_mutexName.isEmpty())
+    {
         QFileInfo fi(*this);
-        mutexname = QString::fromLatin1(MUTEX_PREFIX)
-                    + fi.absoluteFilePath().toLower();
+        m_mutexName = QString::fromLatin1("QtLockedFile mutex ") + fi.absoluteFilePath().toLower();
     }
-    QString mname(mutexname);
+
+    QString mname = m_mutexName;
     if (idx >= 0)
         mname += QString::number(idx);
 
-    Qt::HANDLE mutex;
-    if (doCreate) {
-        QT_WA( { mutex = CreateMutexW(NULL, FALSE, reinterpret_cast<const TCHAR *>(mname.utf16())); },
-               { mutex = CreateMutexA(NULL, FALSE, mname.toLocal8Bit().constData()); } );
-        if (!mutex) {
+    if (doCreate)
+    {
+        const Qt::HANDLE mutex = ::CreateMutexW(NULL, false, reinterpret_cast<const TCHAR *>(mname.utf16()));
```

Comment:
I'm not sure it is a good idea to change FALSE/TRUE with false/true in Win32 API calls. They are of different types. Of course they are currently compatible but this is not guaranteed in any way.

Reply:
You are right, reverted.